dags/atd_service_bot_issues_to_socrata.py

```diff
@@ -0,0 +1,87 @@
+import os
+from datetime import datetime, timedelta
+
+from airflow.models import DAG
+from airflow.operators.docker_operator import DockerOperator
+
+from onepasswordconnectsdk.client import Client, new_client
+import onepasswordconnectsdk
+
+DEPLOYMENT_ENVIRONMENT = os.getenv("ENVIRONMENT", "development")
+ONEPASSWORD_CONNECT_HOST = os.getenv("OP_CONNECT")
+ONEPASSWORD_CONNECT_TOKEN = os.getenv("OP_API_TOKEN")
+VAULT_ID = os.getenv("OP_VAULT_ID")
+
+default_args = {
+    "owner": "airflow",
+    "description": "Publish atd-data-tech Github issues to Socrata",
+    "depends_on_past": False,
+    "start_date": datetime(2015, 12, 1),
+    "email_on_failure": False,
+    "email_on_retry": False,
+    "retries": 1,
+}
+
+docker_image = "atddocker/atd-service-bot:production"
+
+REQUIRED_SECRETS = {
+    "GITHUB_ACCESS_TOKEN": {
+        "opitem": "Github Access Token Service Bot",
+        "opfield": ".password",
+        "opvault": VAULT_ID,
+    },
+    "ZENHUB_ACCESS_TOKEN": {
+        "opitem": "Zenhub Access Token",
+        "opfield": ".password",
+        "opvault": VAULT_ID,
+    },
+    "SOCRATA_API_KEY_ID": {
+        "opitem": "Socrata Key ID, Secret, and Token",
+        "opfield": "add more.apiKeyId",
```

Comment:
I found out that 1PW somewhat awkwardly defines the section as `add more` when you click the `+ Add more` option in an entry without creating a separate section. Not sure what is better - creating a section only to not have `add more` in here or just go with this. 🤷

Reply:
🤮 Yech, that is awkward, I like your description, and I think it's too kind. Thank you for figuring it out! 1PW leaves no clues that I'm seeing.

It handles the `.username` and `.password` "sectionless entries" reasonably enough, I wish they would assume `.apiKeyId` was a top level entry as well. 

Anyway, I'm cool either way.. but I'm leaning towards making a named section in 1PW. My reasoning is that I think the 1Password UI advertising it as being a top level entry without a section, but the API requires that you address it with an express, complete resource designation, and a sure-fire way to make that relationship 1:1 and clear is to use an explicit section in the 1PW GUI instead of the implicit one.

Reply:
cool y'all! i pushed an update with a section and also added a blurb about this in the [GitBook docs](https://app.gitbook.com/o/-LzDQOVGhTudbKRDGpUA/s/-M4Ul-hSBiM-3KkOynqS/1password-connect-server/usage#vault-entry-sections) 🙏